survey_resources/washingtoncommons/washingtoncommons-trip-survey.xml

```diff
@@ -0,0 +1,117 @@
+      <bind jr:preload="timestamp" jr:preloadParams="end" nodeset="/a6dPQCzDbhkycTPXT8os2S/end" type="dateTime"/>
+      <bind nodeset="/a6dPQCzDbhkycTPXT8os2S/What_was_the_main_pu_f_this_trip_trip_leg" required="false()" type="string"/>
+      <bind nodeset="/a6dPQCzDbhkycTPXT8os2S/_2_What_was_the_mode_of_transp" required="false()" type="string"/>
+      <bind nodeset="/a6dPQCzDbhkycTPXT8os2S/Who_was_in_the_vehic_check_all_that_apply" relevant="selected( /a6dPQCzDbhkycTPXT8os2S/_2_What_was_the_mode_of_transp , 'vehicle_alone')" required="true()" type="string"/>
+      <bind jr:preload="uid" nodeset="/a6dPQCzDbhkycTPXT8os2S/meta/instanceID" readonly="true()" type="string"/>
+    </model>
+  </h:head>
+  <h:body>
+    <select ref="/a6dPQCzDbhkycTPXT8os2S/What_was_the_main_pu_f_this_trip_trip_leg">
+      <label>What was the main purpose of this trip/trip leg?</label>
+      <item>
+        <label>Go home</label>
+        <value>go_home</value>
+      </item>
+      <item>
+        <label>Work-related</label>
+        <value>work_related</value>
+      </item>
+      <item>
+        <label>Shopping</label>
+        <value>shopping</value>
+      </item>
+      <item>
+        <label>Recreation</label>
+        <value>recreation</value>
+      </item>
+      <item>
+        <label>Social</label>
+        <value>social</value>
+      </item>
+      <item>
+        <label>Personal business</label>
+        <value>personal_business</value>
+      </item>
+      <item>
+        <label>Pick-up/drop-off/accompany someone</label>
+        <value>pick_up_drop_off_accompany_someone</value>
+      </item>
+      <item>
+        <label>Other</label>
+        <value>other</value>
+      </item>
+      <item>
+        <label>Trip not valid</label>
+        <value>trip_not_valid</value>
+      </item>
+    </select>
+    <select ref="/a6dPQCzDbhkycTPXT8os2S/_2_What_was_the_mode_of_transp">
+      <label>What was the mode of transportation used in this trip?</label>
+      <item>
+        <label>Vehicle</label>
+        <value>vehicle_alone</value>
+      </item>
+      <item>
+        <label>Transit (bus or train)</label>
+        <value>bus</value>
+      </item>
+      <item>
+        <label>On-demand transit (shuttle service)</label>
+        <value>option_12</value>
+      </item>
+      <item>
+        <label>Walk</label>
+        <value>walk</value>
+      </item>
+      <item>
+        <label>Bike</label>
+        <value>bicycle</value>
+      </item>
+      <item>
+        <label>E-bike or e-scooter</label>
+        <value>ebike</value>
+      </item>
+      <item>
+        <label>Ridesharing (Uber/Lyft) or taxi</label>
+        <value>ridesharing_uber_lyft_or_similar_service</value>
+      </item>
+    </select>
+    <select ref="/a6dPQCzDbhkycTPXT8os2S/Who_was_in_the_vehic_check_all_that_apply">
+      <label>Who was in the vehicle? (check all that apply)</label>
+      <item>
+        <label>Myself</label>
+        <value>option_4</value>
```

Comment:
Ditto

Reply:
For both of these, that is the value given in the file I downloaded from the surveys I was sent (it's what I see in the spreadsheet). I could update the spreadsheet myself or circle back with Rey about sending us an updated file.

Reply:
There's actually a lot of this in the onboarding survey as well, will having values like "option_4" limit the data analysis capacities for them?